src/content/docs/setup/configuration/commerce-configuration.mdx

```diff
@@ -109,9 +113,42 @@ Each `key` is described below with links to more details. The `value` for each k
 1. **headers.cs.x-api-key:** Provides access to SaaS storefront services (Catalog Service, Live Search, and Product Recommendations). See [Commerce Services Connector](https://experienceleague.adobe.com/en/docs/commerce-merchant-services/user-guides/integration-services/saas) for details.
 
 </Callouts>
-
 All of the initializer blocks now automatically use any headers for the corresponding block if they are in the config. For example `commerce.headers.cart.Foo` will add a `Foo` header to graphql requests made by the cart dropin. This can be useful if you need to append certain headers for specific requests.
 
+Each `analytics` key, used to instantiate Storefront Instance Context, is described below with links to more details. The `value` for each key is specific to your Commerce instance and can be provided by your Commerce administrator.
+
+For type definitions of the analytics configuration properties, see the [storefrontInstance.ts](https://github.com/adobe/commerce-events/blob/main/packages/storefront-events-sdk/src/types/schemas/storefrontInstance.ts) file in the commerce-events repository.
+
+<Callouts>
+1. **analytics.base-currency-code:** The base currency code for the store. See [Currency Setup](https://experienceleague.adobe.com/en/docs/commerce-admin/stores-sales/site-store/currency/currency-setup) for details.
+
+1. **analytics.environment:** The environment type (e.g., "Production", "Testing"). See [Cloud Environment overview](https://experienceleague.adobe.com/en/docs/commerce-cloud-service/user-guide/project/overview#environment-overview) for details.
+
+1. **analytics.environment-id:** The unique identifier for your Commerce Cloud environment. See [Cloud Environment overview](https://experienceleague.adobe.com/en/docs/commerce-cloud-service/user-guide/project/overview#environment-overview) for details.
+
+1. **analytics.store-code:** The code that identifies your store. See [Step 3: Create stores](https://experienceleague.adobe.com/en/docs/commerce-operations/configuration-guide/multi-sites/ms-admin#step-3-create-stores) for details.
+
+1. **analytics.store-id:** The unique identifier for your store. See [Step 3: Create stores](https://experienceleague.adobe.com/en/docs/commerce-operations/configuration-guide/multi-sites/ms-admin#step-3-create-stores) for details.
```

Comment:
On the linked page, I only see info about web site, name, code, and root category.  Which one does `analytics.store-id` map to?

Reply:
I had a chance to dig into the questions around where to find the `*-id` values for the store, store view and web site entities. Shout out to Russell Albin for clarifying.

These IDs aren't exposed in the Admin UI the same way that other properties are, for example `code` and `name`. Apparently, the easiest way to find the ID is to click through an entity in the Admin UI and look at the URL.

Alternatively, you can find the value in the `app/etc/config.php` file or by running a DB query. However, those alternative methods are not available for ACCS customers.

@sirugh, I saw your internal Slack message this morning asking if we really need the IDs or if we can just rely on code. I think it would be best to get that clarification before proceeding with this PR.

Reply:
Ideally we don't make non-technical users even aware of any of this :D

Partially why I made the tool to automate config generation (as best it could).

The real question we need to resolve is what exact information is necessary to send in analytics, and how do we (or a merchant) get that info?